fileopen/quine

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env/python
```

Comment:
This should be `#!/usr/bin/env python`. With a space between `env` and `python`. @stanzheng